docs/how-to/kubernetes.md

```diff
@@ -219,6 +219,18 @@ Once the Flow is deployed on Kubernetes, you can use all the native Kubernetes t
 
 You can use this to [add or remove replicas](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment), to run [rolling update](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#updating-a-deployment) operations, etc ...
 
+## Integration with Linkerd
+Jina's Kubernetes support comes with a built-in integration for [Linkerd](https://linkerd.io/) as the service mesh for connecting all the Jina services in Kubernetes. All Jina deployment resources are annotated with the necessary [Linkerd annotations](https://linkerd.io/2.11/tasks/adding-your-service/#meshing-a-service-with-annotations). So we recommend every Jina user to [install Linkerd](https://linkerd.io/2.11/getting-started/#step-3-install-the-control-plane-onto-your-cluster) into the Kubernetes cluster they are using Jina in.
+
```

Comment:
This has less to do with your PR and is more of a general question: It seems to me like k8s (and now also service meshes) is really what we recommend to the pros to use, so seems like a really important part of Jina.

Therefore I think that most of the k8s related info should go into the user guide, and quite prominently i would say, instead of being a bit hidden away in this how-to.

Reply:
I would agree @JohannesMessner , any recommendation or suggestion?

Reply:
I would say we either create a page under `Flow/Deploy`, so on the same level as `create a Flow`, or if we have more content we could even create a section on the same level as `Flow` and `Executor`.
![image](https://user-images.githubusercontent.com/44071807/158771073-47464170-65f7-4fd2-a771-043e657ab01d.png)


This can be done independently of this PR, I will create a ticket and discuss with the unofficial docs task force ;) @samsja @cristianmtr

Reply:
I agree with @JohannesMessner we need to highlight the k8s feature in the docs by putting it in the right section

Reply:
Agree, feel free to discuss in the scout of Documentation, but yes agree on the idea

Reply:
very true, the K8s section is not really a how to anymore. But lets do in a separate issue